docs/source/getting_started.rst

```diff
@@ -213,7 +213,7 @@ generic types or your own type aliases), look through the
 
 .. note::
 
-   When adding types, the convention is to import types
+   When adding types, the convention is to import types as std_types
```

Comment:
This is talking about importing `typing` and this *isn't* the convention.

Reply:
Yeah, the convention is "never alias-import stdlib modules unless strictly necessary". Please revert this doc change along with all imports garbled in other files.